apps/portal/controllers/gadgets.jag

```diff
@@ -0,0 +1,103 @@
+<%
+/**
+ * Copyright 2016 WSO2, Inc. (http://wso2.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * All the requests that starts with /gadgets/ will be dealed with this controller
+ */
+var fresh = false;
+
+isDefault = false;
```

Comment:
Where the isDefault is being defined? Is it global variable? Why it's used here?